app-backend/tile/src/main/scala/image/Mosaic.scala

```diff
@@ -267,8 +267,10 @@ object Mosaic {
           val tiles = maybeTiles.flatten
           if (tiles.nonEmpty)
             Option(tiles.reduce(_ merge _))
-          else
-            Option.empty[MultibandTile]
+          else {
+            val cellType = IntUserDefinedNoDataCellType(0)
+            Option(ArrayMultibandTile.empty(cellType, 3, 256, 256))
+          }
```

Comment:
Seems like `IntArrayTile` doesn't have the `alloc` method. I went with the `ArrayTile` type and did alloc with 0 values (cell type of `IntUserDefinedNoDataCellType(0)`). It works, but I'm not sure if that's an optimal solution.